### ISSUE TITLE
Create a PR instead of pushing directly to the main branch

### ISSUE THREAD

**Issue by Panquesito7** [TheAlgorithms/scripts#11]
### Description

Most of the main branches are protected, which means the scripts won't be able to push unless they unprotect the branches, which would be 100% not safe, IMO.
To prevent this, we could create a temporary branch and create a PR to perform the changes without removing branch protection.

**Comment by rohan472000** [comment]
what we have to do in it....do we have to make a script for it???

**Comment by Panquesito7** [comment]
Yes, kind of. @tjgurwara99 and I are working on it (will get back to it soon).
